.github/workflows/test.yaml

```diff
@@ -10,41 +10,44 @@ on:
       - '*'
   pull_request:
   workflow_dispatch:
-    
+
 jobs:
   tests:
-    name: ${{matrix.test}} 
+    name: ${{ matrix.test }}
     runs-on: ubuntu-latest
     strategy:
       matrix:
         test:
-        # TODO:
-        # - browser/patchright
-        # - browser/user_binary
-        # - browser/remote_cdp
-        # - models/openai
-        # - models/google
-        # - models/anthropic
-        # - models/azure
-        # - models/deepseek
-        # - models/grok
-        # - functionality/click
-        # - functionality/tabs
-        # - functionality/input
-        # - functionality/scroll
-        # - functionality/upload
-        # - functionality/download
-        # - functionality/save
-        # - functionality/vision
-        # - functionality/memory
-        # - functionality/planner
-        # - functionality/hooks
-        - test_controller
-        - test_tab_management
-        - test_sensitive_data
+          - browser/patchright
+          - browser/user_binary
+          - browser/remote_cdp
+          - models/openai
+          - models/google
+          - models/anthropic
+          - models/azure
+          - models/deepseek
+          - models/grok
+          # - functionality/click
+          # - functionality/tabs
+          # - functionality/input
+          # - functionality/scroll
+          # - functionality/upload
+          # - functionality/download
+          # - functionality/save
+          # - functionality/vision
+          # - functionality/memory
+          # - functionality/planner
+          # - functionality/hooks
+          - test_controller
+          - test_tab_management
+          - test_sensitive_data
+
     steps:
-      - uses: actions/checkout@v4
-      - uses: astral-sh/setup-uv@v6
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Setup uv with cache
+        uses: astral-sh/setup-uv@v6
```

Comment:
naming the steps is *less* clear not more clear, why make the viewer click to expand when you could just see at a glance *exactly* what it's doing in each step